README.md

```diff
@@ -15,6 +15,198 @@ For light-way tests with smaller Samples, where using separate JSON-files might
+
+   @PrepareSampler
+   private PersonDao personDaoSampler;
+   ...
+}
+```
+
+Inside of a concrete test method we would now use the Sampler to define a stub:
+```
+@Test
+void greetingShouldBeGenerated() {
+   Sample.of(personDaoSampler.loadPerson(1)).is(new Person("Sarek"));
+   ...
+}
+```
+This means, the method `PersonDao::loadPerson` will be stubbed. If `loadPerson` is called with the parameter
+`1` the stub will return the __Sample__ `new Person("Sarek")`. Otherwise, the original method will be called.
+The stub applies to all instances of `PersonDAO`, no matter where the instances occur. 
+It will be active for the lifetime of the test method, so each test method can declare its own stubs. 
+
+That's it, we can now call the testee and check the results as usual:
+
+```
+@Test
+void greetingShouldBeGenerated() {
+   Sample.of(personDaoSampler.loadPerson(1)).is(new Person("Sarek"));
+
+   assertEquals("Hello Sarek!", greetingService.createGreeting(1));
+}
+```
+
+__[TLDR;]__ Just to see that the stub is actually doing something, we could clear all stubs and repeat the
+assertion. Now the original method is used again, and we get another return value:
+
+```
+void greetingShouldBeGenerated() {
+   Sample.of(personDaoSampler.loadPerson(1)).is(new Person("Sarek"));
+
+   assertEquals("Hello Sarek!", greetingService.createGreeting(1));
+
+   Sampler.clear();
+
+   assertEquals("Hello Geordi La Forge!", greetingService.createGreeting(1));
+}
+```
+
+### Define a Sampler by API
+If you want to use DeepSampler outside of JUnit or don't want to use the annotations, it is possible to create a Sampler
+using the API:
+```
+PersonDao personDaoSampler = Sampler.prepare(PersonDao.class);
+```
+
+### SamplerFixtures
+Usually it will not be enough to write just one test for a particular compound like the `GreetingService`, so it
+would be tedious to repeat the definition of stubs in each test case. To ease that, `SamplerFixtures` can be used to 
+define reusable stubs. You can also think of a SampleFixutre as a definition of a test compound, because the stubs
+would isolate the compound from the environment in order to make it testable: 
+```
+public class GreetingServiceCompound implements SamplerFixture {
+   @PrepareSampler
+   private PersonDao personDaoSampler;
+
+   @Override
+   public void defineSamplers() {
+      Sample.of(personDaoSampler.loadPerson(Matchers.anyInt()));
+   }
+}
+```
+Once we have such a SamplerFixture, we can bind it to a test case using the annotation `@UseSamplerFixture` like so:
+```
+@Test
+@UseSamplerFixture(GreetingServiceCompound.class)
+void recordSamplesToJson() {
+   ...
+}
+```
+The stubs, that are defined by the SamplerFixture, are now active for the time the method runs. 
+
+You might have noticed that the stub doesn't define a concrete Sample anymore. We have done this, because we
+rather want to read the Samples from a JSON-file now. How this can be achieved is explained by the next two chapters.
+
```

Comment:
Maybe you could add the following part as subchapter

Ex.: 
### File-Usage
#### Record a JSON-Sample
...
#### Load of a JSON-Sample